ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake

```diff
@@ -27,18 +32,35 @@
 # @public
 #
 macro(ament_auto_find_build_dependencies)
-  if(ARGN)
+  cmake_parse_arguments(ARG "" "" "REQUIRED_PACKAGES" ${ARGN})
```

Comment:
thanks. https://github.com/ament/ament_cmake/commit/0e178fee5a415caf30f11ba083ee3c5bb2de914e prefixes the variable